### DISCUSSION TITLE
Enabling auto dereferencement for pointers

### DISCUSSION THREAD

**Discussion by b-kitt** [microsoft/vscode-cpptools#7321] [Q&A] [answered]
https://github.com/microsoft/vscode-cpptools/issues/862

I just downloaded the latest version of VScode and installed the extension, but I can't get auto dereferencement to work.

Is there a setting I need to modify to get it to work?

For instance when I have

```
vector<string> * msg2;
msg2.
```

and hit  ```ctrl + space``` nothing comes up until I modify the ```.``` to a ```->```.

**Comment by sean-mcmanus** [comment, marked as answer]
It's working for me. What version do you have?

![image](https://user-images.githubusercontent.com/19859882/114113965-a7a32200-9894-11eb-991e-5f0359d12855.png)

Can you provide a screenshot? You may have another extension installed that is intereferring with ours?

**Comment by b-kitt** [comment]
You were correct, it was one of the extensions that was suggested when I installed VScode:

C++ Intellisense
austin.code-gnu-global
